GOVERNANCE.md

```diff
@@ -1,132 +1,175 @@
 - Providing moral support (a ‘thank you’ goes a long way)
-- Writing tutorials
+- Writing tutorials to contextualize OpenRefine for your community.
 
 How to become an OpenRefine user? [Download OpenRefine](http://openrefine.org/download.html) and start refining!
 
 ### Contributors
 Contributors are users getting involved in the project more closely. Contributions can take many forms:
 
 - Supporting new users via the [user discussion list](https://forum.openrefine.org/c/support/12)
-- Submitting patches to fix bugs or add features via pull requests
+- Submitting patches to fix bugs or add features via pull requests, comment on pull requests and help other contributors via the [developer discussion list](https://forum.openrefine.org/c/dev/8)
 - Maintaining and improving the [website](https://openrefine.org/)
 - Writing and maintaining the [documentation](https://github.com/OpenRefine/openrefine.org/tree/master/docs)
+- Helping [design interfaces](https://openrefine.org/docs/technical-reference/introduction) for OpenRefine
 
 How to become an OpenRefine contributor? You will find more details in our [contributing guideline](https://github.com/OpenRefine/OpenRefine/blob/master/CONTRIBUTING.md)
 
 ### Committers
-If you make regular contribution to OpenRefine, you will most likely become a Committer.
+If you make regular contributions to OpenRefine, you will most likely become a Committer.
 
-Committers have earned enough trust from the community to review and merge pull requests.
+Committers have earned enough trust from the community to
 
-Therefore Committers:
-- Help contributors via the [developer discussion list](https://forum.openrefine.org/c/dev/8).
 - Triage issues, pull requests and [projects](https://github.com/OpenRefine/OpenRefine/projects)
 - Have direct access to the code base to create new branches
-- Organize the wiki
-- Review and merge pull requests submitted by contributors
+- Organize the GitHub wiki
+- Review pull requests submitted by contributors
+- Run the CI pipeline on your PRs automatically
 - Are part of the OpenRefine organization and have the OpenRefine badge on their GitHub profile
 
-#### How to become a Committer?  
+**How to become a Committer?**: Committers are voting-binding members; they select and elect new members through a voting process. Nominations should be submitted to the [developer discussion list](https://forum.openrefine.org/c/dev/8), where contributors can nominate themselves or other contributors. See the Member Election section below for more details.
+
+**Current list of Committers**: The list is available here: https://github.com/orgs/OpenRefine/people.
+
+### Core Developer Group
+Also referred to as Core Dev. The Core Developer Group are trusted developers who
+- Merge pull requests
+- Publish releases
+- Review security vulnerability reports
+- Provide long-term project vision with the Advisory Committee team
```

Comment:
```suggestion
- Provide long-term project vision with the community
```

This is done via public discussions on the forum, on github, using various user survey (see the [feature survey 2024](https://forum.openrefine.org/t/results-from-the-feature-prioritization-survey-2024/1847) for example) and roadmap exercise (see [barcamp notes](https://forum.openrefine.org/t/openrefine-2024-barcamp-strategizing-openrefine-roadmap/1633)) or [proposed roadmap](https://forum.openrefine.org/t/prioritisation-of-wikimedia-sveriges-development-work/1803)